dotcom-rendering/src/components/Dropdown.tsx

```diff
@@ -35,6 +35,7 @@ interface Props {
 	dataLinkName: string;
 	cssOverrides?: SerializedStyles;
 	children?: React.ReactNode;
+	defaultIsExpanded?: boolean;
```

Comment:
Is this done in order to simulate user interaction?

I’d recommend using `@testing-library/react`, as we do in [`AustralianTerritorySwitcher`](https://github.com/guardian/dotcom-rendering/blob/2ed9dd5f731246a87274421dece72961341fa96f/dotcom-rendering/src/components/AustralianTerritorySwitcher.importable.stories.tsx#L17-L24) and [`Nav`](https://github.com/guardian/dotcom-rendering/blob/2ed9dd5f731246a87274421dece72961341fa96f/dotcom-rendering/src/components/Nav/Nav.stories.tsx#L123-L133)

Reply:
Ooh interesting, can I use that in the context of a story? I'm particularly interested in having a Chromatic snapshot for this component in the expanded state as there's been a visual regression here (which I think Chromatic would have picked up).

Reply:
Yes, it’s a pretty useful feature: https://storybook.js.org/docs/writing-tests/interaction-testing

Reply:
👍 Nice, thanks. I'll investigate.